src/Common/Infrastructure/ApiController.cs

```diff
@@ -7,60 +7,17 @@ namespace Common.Infrastructure
     using System.Net;
     using System.Threading;
     using System.Threading.Tasks;
-    using Api.Infrastructure;
     using Be.Vlaanderen.Basisregisters.Api;
     using Be.Vlaanderen.Basisregisters.Api.Exceptions;
     using Be.Vlaanderen.Basisregisters.AspNetCore.Mvc.Middleware;
-    using Microsoft.AspNetCore.Http;
-    using Microsoft.AspNetCore.Http.Headers;
+    using Extensions;
     using Microsoft.AspNetCore.Mvc;
-    using Microsoft.AspNetCore.Mvc.Abstractions;
-    using Microsoft.AspNetCore.Mvc.Infrastructure;
     using Microsoft.Extensions.Logging;
     using Newtonsoft.Json;
     using RestSharp;
     using StackExchange.Redis;
 
-    public abstract class PublicApiController : ApiController
-    {
-        public static string DetermineAndSetContentFormat(
-            string format,
-            IActionContextAccessor actionContextAccessor,
-            HttpRequest request)
-        {
-            format = !string.IsNullOrWhiteSpace(format)
-                ? format
-                : actionContextAccessor.ActionContext.GetValueFromHeader("format")
-                  ?? actionContextAccessor.ActionContext.GetValueFromRouteData("format")
-                  ?? actionContextAccessor.ActionContext.GetValueFromQueryString("format");
-
-            var acceptType = request.GetTypedHeaders().DetermineAcceptType(format, actionContextAccessor.ActionContext.ActionDescriptor);
-            var contentType = acceptType.ToMimeTypeString();
-
-            request.Headers[HeaderNames.Accept] = contentType;
```

Comment:
Since this seems to be gone, where do you set it now? It needs to be set to have ProblemDetails work correctly. All the MVC bits downstreams need this header to do content negotation in their own bits.

Reply:
Function got inlined 
https://github.com/Informatievlaanderen/public-api/pull/76/files#diff-f8dfb413c45f1e5378d9c7bdf3b10a7aL31

Reply:
Maybe make a ContentFormat.Set extension method which sets it again? :)

Reply:
Yes, but only inlined for the api key auth attribute. All the other cases need it too. Other option is you make a filter and apply it to base api controller which sets it on every incoming request?

Reply:
Added SupportUrlFormat attribute